CMakeLists.txt

```diff
@@ -1136,18 +1136,9 @@ endif ()
 ## 3RD PARTY
 ################################################################################
 
-add_definitions(-DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX=1) #fix random provider
-add_definitions(-DBOOST_ALL_NO_LIB=1) #disable boost autolink on windows
-add_subdirectory(3rdParty)
-link_libraries(
-  absl::flat_hash_set
-  absl::flat_hash_map
-  absl::node_hash_set
-  absl::node_hash_map
-  absl::synchronization
-  absl::strings
-)
-
+add_definitions(-DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX=1)  # fix random provider
+add_definitions(-DBOOST_ALL_NO_LIB=1)  # disable boost autolink on windows
+add_subdirectory(3rdParty EXCLUDE_FROM_ALL)
```

Comment:
Note: previously only some subdirectories in `3rdParty` were marked with `EXCLUDE_FROM_ALL`, now it is going to be everything in `3rdParty`.
I guess the `EXCLUDE_FROM_ALL` tag is useful for IDE users only, as dependencies will be handled correctly regardless of the `EXCLUDE_FROM_ALL` anyway.

Reply:
Yes, I made this because noticed that some directory which needs exclude from all forgot it.

abseil, fmt, few others